Solutions/FlareSystemsFirework/Workbooks/FlareSystemsFireworkOverview.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Hi @aviau, 

The expectation for workbooks is that they contain at least 4 charts/graphs.

Could you add at least 2 more please?

Reply:
Alright, done!